bin/lib/common.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs')
+const extend = require('extend')
+const { cyan, bold } = require('colorette')
+const util = require('util')
+
+module.exports = {}
```

Comment:
I could be wrong, but isn't this the default behaviour in CommonJS?

Reply:
Yeah, you're right, it's unnecessary. I'll remove it.